### PR TITLE
My attempt

### PR DESCRIPTION
**Pull request by dickyince**

### CODE REVIEW

**Comment by a-panayotopoulos** [comment]
Ruby?! :gem:

File: rpi/strategy/data.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Data
+  attr_reader :stuff
+  attr_accessor :formatter
+
+  def initialize( formatter )
+    @stuff = { title, 'The Title' }
+    @formatter = formatter)
```
Comment (Dave-McCraw) [inline rpi/strategy/data.rb]: Do brackets not need to be matched? This shit is crazy!

File: rpi/strategy/data.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Data
+  attr_reader :stuff
+  attr_accessor :formatter
+
+  def initialize( formatter )
+    @stuff = { title, 'The Title' }
```
Comment (Utopia1138) [inline rpi/strategy/data.rb]: I think this should be initialized like:
```ruby
@stuff = { title: 'The Title' }
```

File: rpi/strategy/data.rb (hunk 1 above)
Comment (Utopia1138) [inline rpi/strategy/data.rb]: Syntax error! :boom:

File: rpi/strategy/data.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class Data
```
Comment (Utopia1138) [inline rpi/strategy/data.rb]: It looks like class `Data` is actually a core ruby abstract type - when I try and run this I get an error attempting to allocate it `new': allocator undefined for Data (TypeError)` - if I rename it `MyData` (and fix the other syntax errors) it works.

**Pull request merged**